sdks/python/container/base_image_requirements_manual.txt

```diff
@@ -32,14 +32,11 @@ cython<1
 # some versions of libraries that launch Beam pipelines, like tensorflow-transform.
 # Leaving 'future' in our containers for now prevent breaking tft users.
 future
-# TODO: Remove the upper bound once Tensorflow 2.11 is released.
-# https://github.com/apache/beam/issues/23355
-google-cloud-profiler<4.0.0
```

Comment:
Any reason removing google-cloud-profiler dep in base image? This has broken the google-cloud-profiler support on Beam v2.47.0

Reply:
This was not intentional. Looks like Dataflow containers still include it.

Reply:
we intentionally removed google-cloud-debugger from Dataflow containers, perhaps there was confused between this two deps.

Reply:
Dataflow container now does not include it. Can be seen by https://github.com/apache/beam/blob/master/sdks/python/container/py38/base_image_requirements.txt where google-cloud-profiler is no longer listed in. I now get error using profiler: #26698

Reply:
ahh...this was not intentional. 

@Abacn these are beam external images. Dataflow images are internal: https://source.corp.google.com/piper///depot/google3/third_party/cloud/dataflow/worker/images/base_python/Dockerfile.base-python;rcl=522192081;l=117

Reply:
Thanks, this is less severe, so it only affects dev version